packages/mobile/src/recipients/RecipientPicker.tsx

```diff
@@ -56,6 +66,29 @@ function RecipientPicker(props: RecipientProps) {
   const [isKeyboardVisible, setKeyboardVisible] = useState(false)
   const [isSendToAddressWarningVisible, setSendToAddressWarningVisible] = useState(false)
 
+  const kitCall = useAsync(async () => {
```

Comment:
Hmm, this looks a bit weird, can we simply call `await getContractKitAsync(false)` inside of the other `useAsync`?

Reply:
I guess we can, I wanted to avoid calling `getContractKitAsync` several times. However, if the contract kit was already created, calling `getContractKitAsync` is not that expensive. I will change it :)